jenkins-job-builder/config/definitions/jjb.yml

```diff
@@ -1,6 +1,6 @@
 - job:
     name: jenkins-job-builder
-    node: (small && trusty) || master
+    node: small && trusty
```

Comment:
they are manually disabled. And you are right, they take for ever. We should just get them removed or commented out "officially"